README.md

```diff
@@ -270,6 +270,18 @@ buf {
 }
 ```
 
+You can optionally specify a different `buf.gen.yaml` file as well:
```

Comment:
This should be under a separate heading from "Generating Dependencies" and added to the table of contents. Possibly called "Further Configuration".

Reply:
Not something for this PR: it seems like a potential normal use case is to want several invocations of `buf generate`, each with a different template. I'll look into that use case separately.